controllers/clustergroupupgrade_controller.go

```diff
@@ -1193,19 +1196,28 @@ func (r *ClusterGroupUpgradeReconciler) updateConfigurationPolicyHubTemplate(
 }
 
 func (r *ClusterGroupUpgradeReconciler) updateConfigurationPolicyName(
-	clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade, metadata interface{}) {
+	clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade, metadata interface{}) error {
 
 	metadataContent := metadata.(map[string]interface{})
 	name := utils.GetResourceName(clusterGroupUpgrade, metadataContent["name"].(string))
-	safeName := utils.GetSafeResourceName(name, clusterGroupUpgrade, utils.MaxPolicyNameLength, 0)
+	safeName, err := utils.GetSafeResourceName(name, clusterGroupUpgrade, utils.MaxPolicyNameLength, 0)
+	if err != nil {
+		return err
```

Comment:
do you think wrapping these errors is useful?

Reply:
No need since I removed the error in NewSafeResourceName.